lab5.cpp

```diff
@@ -0,0 +1,23 @@
+﻿#include "main.h"
+
+
+int main()
+{
+    int hrs, min, res;
+    int sec, my_sec;
+
+    int* arr = new int[3];
```

Comment:
no one knows what new int means [3]

Reply:
https://www.guru99.com/cpp-dynamic-array.html#:~:text=It's%20easy%20to%20initialize%20a,be%20added%20to%20the%20array